KARL_CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+Fix: Update `background-color` styleguide.
```

Comment:
Un petit `-` ?

Reply:
Et je changerais `styleguide` par `on the styleguide` pour faire une phrase un peu plus précise.